Modules/Common/include/Common/TH1Ratio.inl

```diff
@@ -168,6 +168,29 @@ void TH1Ratio<T>::update()
       }
     }
     T::Divide(mHistoDen);
+    if (mSumw2Enabled) { // Re-calculate weighted errors; taken from https://root.cern.ch/doc/master/classTH1.html#ae7c34d74d043ff9d6816e5b026f85133
```

Comment:
@f3sch in I understand correctly, you found that `T::Divide()` does not compute correctly the errors, even if `Sumw2()` was called beforehand for all the histograms? Do you understand why?

Reply:
Hi @aferrero2707, you are right Divide already does the right thing for the weighted errors, but there is no option to instead do binomial error propagation. I changed it accordingly.

Reply:
Thanks @f3sch! Have you already double-checked that `Divide()` really computes the errors correctly? I have to admit I just assumed it was ok...

Reply:
@aferrero2707, yes I did.

Reply:
@f3sch then I am wondering if it would be better to use the ROOT functions for the binomial errors as well.
Instead of `T::Divide(mHistoDen);` one could call `T::Divide(mHistoNum, mHistoDen, 1, 1, option);`, and set `option` to `"b"` if binomial errors are requested. 

In this case `T::Add(mHistoNum);` would be needed only in the uniform scaling case.

Reply:
Done.

Reply:
@f3sch if you have a test set-up available, I'd recomment to check that ROOT is really doing what you expect for the binomial errors, just to be 100% sure that your custom code is really not needed. Thanks a lot!

Reply:
@aferrero2707, the plots look identical to ones I have posted above.

Reply:
Very good! I have no more comments.

Reply:
thanks for your comments!